docs/example/example/processes/template_py_process.py

```diff
@@ -0,0 +1,277 @@
+"""Describe what tasks this Process accomplishes."""
+# Order of imported packages should be:
+# future (Python 2)
+# Standard libraries
+from pathlib import Path
+from shutil import copy2
+
+# Third party libraries
+from plumbum import TEE
+
+# resolwe
+from resolwe.process import (
+    Process,
+    SchedulingClass,
+    Persistence,
+    Cmd,
+    StringField,
+    TextField,
+    BooleanField,
+    IntegerField,
+    FloatField,
+    DateField,
+    DateTimeField,
+    UrlField,
+    SecretField,
+    FileField,
+    FileHtmlField,
+    DirField,
+    JsonField,
+    ListField,
+    DataField,
+    GroupField,
+)
+
+# In-house libraries (i.e. resolwe_bio)
+# Local libraries
+
+
+class MergeReads(Process):
+    """Process description in one line ends with a period.
+
+    Continue with a more elaborate description of what the process
+    does, short layout of the algorithm and what output(s) to expect.
+
+    Note that this is not a working process, but rather just shows
+    bits and pieces that can be used in your real process.
+    """
+
+    slug = "merge-reads"
+    name = "Merged reads"  # human readable name of the process
+    process_type = "data:reads:merged:"
+    version = "1.0.0"
+    # See a list of existing categories in resolwe-bio documentation
+    # https://resolwe-bio.readthedocs.io/en/latest/catalog-category.html
+    category = "Read Processing"
+    # Dynamically generated name of the resulting datum object. Lookup
```

Comment:
I think it is more common to name the object "Data object", at least in Resolwe.

Reply:
How does this sound?

```
data_name = 'Data object ({{ string_field.file|default("?") }})'
```